Eli250/my-brand
# hqwp5RU1 Contact and signin page

###What does this PR do?

 Create signin and admin panel

###Description of Task to be completed?

This PR creates the contact and sigin page for the admin panel.

###How should this be manually tested?

Check if all text inputs are provided and validated
Entering wrong input an check if the system shows error message
Enter credential and check if you reach the admin page
###Relevant pivotal tracker stories?

hqwp5RU1

###Screenshots
![Screenshot from 2021-12-24 18-43-21](https://user-images.githubusercontent.com/60324751/147365224-954ad875-fa20-4a9e-8af7-ac9c088d1819.png)
![Screenshot from 2021-12-24 18-43-26](https://user-images.githubusercontent.com/60324751/147365234-e99415c9-cee0-4b7b-a6d6-949854268706.png)
![Screenshot from 2021-12-24 18-43-30](https://user-images.githubusercontent.com/60324751/147365235-5a2d0870-c08d-4a27-9c4f-3f0b96472856.png)
![Screenshot from 2021-12-24 18-43-43](https://user-images.githubusercontent.com/60324751/147365237-02093da9-02b0-4ac3-8ee9-2d3a65504915.png)
